doc/build-instructions/Build_Instructions_V20.md

```diff
@@ -758,17 +758,15 @@ bash configure --openjdk-target=${OPENJ9_CC_PREFIX} \
                --with-x=${OPENJ9_CC_DIR}/${OPENJ9_CC_PREFIX}/ \
                --with-freetype-include=${OPENJ9_CC_DIR}/${OPENJ9_CC_PREFIX}/libc/usr/include/freetype2 \
                --with-freetype-lib=${OPENJ9_CC_DIR}/${OPENJ9_CC_PREFIX}/libc/usr/lib \
-               --with-freemarker-jar=/root/freemarker.jar \
-               --with-boot-jdk=/root/bootjdk17 \
-               --with-build-jdk=/root/bootjdk17 \
-               --with-cmake=no \
+               --with-boot-jdk=/root/bootjdk18 \
+               --with-build-jdk=/root/bootjdk19 \
```

Comment:
The build JDK needs to be the current version (20).

Reply:
Updated.